Updated Fusion licensing (#717)
seqeralabs/docs: fusion_docs/licensing.md

### For Seqera Enterprise instances without network connectivity, and direct Fusion use with Nextflow  

- Run Fusion by using a Cloud API via Nextflow’s `TOWER_ACCESS_TOKEN` environment variable in the `nextflow.config` file.

> - This will count toward the Free/Paid usage associated with your Cloud organization.

Reviewer: ```suggestion
- Throughput counts towards the free or paid usage associated with your Cloud organization.
```
I find starting the bullet point with "This" to be unclear. Suggestion to rephrase to make it clear it's the throughput that is being measured.
